packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -17,6 +17,14 @@ interface IDependencySet {
     function isInDependencySet(uint256 _chainId) external view returns (bool);
 }
 
+/// @title IL1Block
+/// @notice Interface for L1Block with only `isDeposit()` method.
+interface IL1Block {
+    /// @notice Returns whether the call was triggered from a a deposit or not.
+    /// @return True if the current call was triggered by a deposit transaction, and false otherwise.
+    function isDeposit() external view returns (bool);
+}
+
```

Comment:
Don't we have the interface elsewhere?
is this common practice on the OP repo?

Reply:
Is a practice they use, because of the different versions of the contracts.
You can see this practice here as reference: https://github.com/ethereum-optimism/optimism/blob/051db548a7f968b8c1b88da7135d0bf9cbea4954/packages/contracts-bedrock/src/L2/CrossL2Inbox.sol#L12

Reply:
Sweet, let's go then!